File: user-service/src/main/java/com/example/userservice/dto/UserLogRequest.java
package com.example.userservice.dto;

import lombok.*;

@Getter
@Setter
@ToString
public class UserLogRequest {
    private String userId;
    private String requestData;
    private String requestMs;
    private String requestSource;
    private Integer requestNum;

    @Builder
    private UserLogRequest(String userId, String requestData, String requestMs, String requestSource, Integer requestNum) {
        this.userId = userId;
        this.requestData = requestData;
        this.requestMs = requestMs;
        this.requestSource = requestSource;
        this.requestNum = requestNum;
    }
}

File: pod-service/src/main/kotlin/miniproject.kubeBoard.podservice/entity/pod/req/UserLogRequest.kt
package miniproject.kubeBoard.podservice.entity.pod.req

import lombok.Builder
import lombok.Data
import lombok.Getter
import lombok.Setter

@Getter
@Setter
@Builder
data class UserLogRequest(
        val userId: String,
        val requestMs: String,
        val requestData: String,
        val requestSource: String,
        val requestNum: Int,
)